.expeditor/config.yml

```diff
@@ -21,36 +21,39 @@ github:
   # allow bumping the major release via label
   major_bump_labels:
     - "Expeditor: Bump Version Major"
-  # Which Github branches to build releases from, and what versions
-  # (as determined by the value in the VERSION file) those branches are responsible
-  # for building.
-  release_branch:
-    - master:
-        version_constraint: "*"
+
+# Which Github branches to build releases from, and what versions
+# (as determined by the value in the VERSION file) those branches are responsible
+# for building.
+release_branches:
+  - master:
```

Comment:
If `ffi` is their default branch, we should change this from `master` to `ffi` for them.